snapcraft/plugins/make.py

```diff
@@ -94,9 +94,7 @@ def __init__(self, name, options, project):
         super().__init__(name, options, project)
         self.build_packages.append('make')
 
-    def build(self):
-        super().build()
-
+    def _build(self, env):
```

Comment:
Thanks for the quick response. Do you mean changing the name to something like `make`? I've pushed up a very small commit to update the name and usage. Let me know what else might need changing.